lib/compiler/src/engine/inner.rs

```diff
@@ -200,12 +200,38 @@ impl Engine {
         Ok(Arc::new(Artifact::deserialize(self, bytes)?))
     }
 
+    /// Deserializes a WebAssembly module
     #[cfg(not(target_arch = "wasm32"))]
+    pub fn deserialize_checked(&self, bytes: &[u8]) -> Result<Arc<Artifact>, DeserializeError> {
+        #[cfg(not(target_arch = "wasm32"))]
+        {
+            Ok(Arc::new(Artifact::deserialize_checked(self, bytes)?))
+        }
+        #[cfg(target_arch = "wasm32")]
```

Comment:
how can this `cfg` be used, the function has the oposite config.

Reply:
Thanks!

This was just an artifact from reverting the renaming.